master/buildbot/test/fake/latent.py

```diff
@@ -156,7 +163,7 @@ def patchBot(self, case, remoteMethod, patch):
         case.patch(BotBase, remoteMethod, patch)
 
 
-class ControllableLatentWorker(AbstractLatentWorker):
+class ControllableLatentWorkerMixin(object):
```

Comment:
Inheriting from `object` is not required anymore in python 3. Can you remove this?

Reply:
Whoops, the commit is from before we dropped python 2. Fixed now.